### DISCUSSION TITLE
How to connect anki to use saladict word search history

### DISCUSSION THREAD

**Discussion by bheeshmpita** [crimx/ext-saladict#2014] [Q&A]
HI, I have read the tutorial for connecting anki with saladict. So, I have installed anki with anki connect addon and enabled the same from saladict settings. But I am lacking somewhere in the saladict settings to make it work. Help me by guiding for the same. What I want to achieve is the word search history from the saladict gets synced to the anki automatically (or manually ) whenever I open anki.
Thanks

**Comment by crimx** [comment]
Hi, word search history cannot be synced to Anki.

**Reply by bheeshmpita** [reply]
ok, what is the function for anki connect in saladict settings. Secondly, how do i export word search history with its contents as word, meaning, example sentence to anki. I can see the export button in word search history, how to export in the required format?

**Reply by crimx** [reply]
You can change the Saladict export template to something like this ```%text% ` %context% ` %trans% ` %note%```. In Anki, set the text import separator to ``` ` ```.

**Reply by bheeshmpita** [reply]
> You can change the Saladict export template to something like this

Where should I do this, i am unable to locate appropriate settings for this

and what is the function of anki connect in saladict if not word search history?

**Reply by crimx** [reply]
Anki Connect is for words in Notebook, as it's under the "Notebook" configurations.

**Reply by bheeshmpita** [reply]
and for template settings where should I visit?

**Reply by crimx** [reply]
There is an "export" button on top of the Notebook page or History page. On the popup export panel, you can edit the template on the left side and see the generated text on the right side.
